src/statement/types.rs

```diff
@@ -202,3 +202,22 @@ unsafe impl<'a> OdbcType<'a> for f64 {
     fn column_size(&self) -> ffi::SQLULEN { size_of::<Self>() as ffi::SQLULEN }
     fn value_ptr(&self) -> ffi::SQLPOINTER { self as *const Self as ffi::SQLPOINTER }
 }
+
+unsafe impl<'a> OdbcType<'a> for Vec<u8> {
+    fn sql_data_type() -> ffi::SqlDataType { ffi::SQL_VARCHAR }
```

Comment:
This should rather be `SQL_VARBINARY`, so no encoding is assumed. It's a design error to have the Data Type being part of tied trait anyway, but for now, I would just change this to `SQL_VARBINARY` to prevent the driver and driver manager from performing transformations on the data send through `Vec<u8>`.